docs/docs-source/docs/modules/get-started/pages/define-avro-schema.adoc

```diff
@@ -5,125 +5,58 @@
 
 include::ROOT:partial$include.adoc[]
 
-Let's start building the avro schema for the domain objects that we need for the application. These schema files will have an extension `.avsc` and will go directly under `src/main/avro` in the project structure that we discussed earlier.
+Let's start building the avro schema for the domain objects that we need for the application. 
+These schema files have the extension `.avsc` and go directly under `src/main/avro` in the project structure that we discussed earlier.
 
 In the Wind Turbine example, we will use the following domain objects:
 
 * **SensorData:** The data that we receive from the source and ingest through our ingress.
 * **Measurements:** A substructure of `SensorData` that abstracts the values of interest, such as wind speed.
-* **Metric:** A domain object that identifies the Turbine we would like to track and measure.
-* **InvalidMetric:** An object that abstracts the erroneous metric.
+* **Metric:** A domain object that identifies the reported values we would like to track and measure.
+* **InvalidMetric:** An object that encapsulates the erroneous metric.
 
-During the build process, the Cloudflow plugin system processes the schemas. For each of these schema files, Cloudflow will generate Scala case classes that can be directly used from within the application.
+During the build process, the Cloudflow plugin system processes the schemas. 
+For each of these schema files, Cloudflow will generate Scala case classes that can be directly used from within the application.
```

Comment:
Scala or Java classes ?

Reply:
In the context of the getting-started, these are Scala 'case classes'.